test/DebugInfo/Generic/undef-func-call.ll

```diff
@@ -0,0 +1,98 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o - | llvm-dis -o %t.ll
+; RUN: llc -mtriple=%triple < %t.ll
+
```

Comment:
I think the tests should have CHECK lines, or an explicit comment that this is a regression test for crash reported in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/524

Reply:
Actually I just wanted to check that translator doesn't crash in this pipeline. Will add a comment (and remove llc run as a redundant run)